spec/spec_helper.rb

```diff
@@ -27,17 +27,17 @@
 require "rspec/longrun"
 #require "narray/narray"
 
-require "../lib/nmatrix"
-require "../lib/nmatrix/rspec"
+require "./lib/nmatrix"
+require "./lib/nmatrix/rspec"
```

Comment:
Thanks @abinashmeher999 . It worked :+1: 

@mohawkjohn I reverted all useless changes from spec directory. It's in jruby_test_rebase branch. A few commits still need need to be reverted for files in lib directory.